ui/components/app/confirm-data/confirm-data.js

```diff
@@ -38,28 +45,33 @@ const ConfirmData = ({ txData, dataComponent }) => {
     : '';
 
   return (
-    <Box color={Color.textAlternative} className="confirm-data" padding={4}>
-      <Box paddingBottom={3} paddingTop={2}>
-        <Text
-          as="span"
-          textTransform={TextTransform.Uppercase}
-          variant={TextVariant.bodySm}
-        >
-          {`${t('functionType')}:`}
-        </Text>
-        <Text
-          as="span"
-          color={Color.textDefault}
-          paddingLeft={1}
-          textTransform={TextTransform.Capitalize}
-          variant={TextVariant.bodySmBold}
-        >
-          {`${functionType} ${functionParams}`}
-        </Text>
+    <Box>
+      <Box marginLeft={4} marginRight={4} marginTop={4}>
+        <TransactionInsightsDeprecationAlert />
+      </Box>
+      <Box color={Color.textAlternative} className="confirm-data" padding={4}>
```

Comment:
[nit] we can move `className="confirm-data" to the parent Box element to follow the pattern we have in our codebase. This would also require us to update one of the styles

`& > .disclosure {` → `& .disclosure {` (or specifying a new classname)

Reply:
This is existing div only wrapped in a parent element and an alert placed on top of it. I would avoid refactoring it and functionality is being deprecated.

Reply:
Yeah, I don't think there is much reason to spend time maintaining this if we are gonna remove it in a couple of releases.